Lab1/Lab1/Form1.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Drawing;
+using System.Windows.Forms;
+
+using Geometry;
+using Geometry.ShapeList;
+
+namespace Lab1
+{
+    public partial class MainForm : Form
+    {
+        private Shapes _shapes;
+
+        public MainForm()
+        {
+            this._shapes = new Shapes()
```

Comment:
Even if you can't create the object of an abstract class you can create the collection of generics with this class so instead of your shapes realization you can write new List\<Shape\>() and work with it as your Shapes class